app/services/sign_in/access_token_jwt_encoder.rb

```diff
@@ -29,7 +29,15 @@ def payload
         anti_csrf_token: access_token.anti_csrf_token,
         last_regeneration_time: access_token.last_regeneration_time.to_i,
         version: access_token.version
-      }
+      }.merge(user_attributes)
```

Comment:
I would make `user_attributes` an actual key here, and in a nested hash as the values are the user attributes:
`user_attributes: { first_name: blah, last_name: blah, email: blah }`

Reply:
If access token takes a `user_attributes` hash then we could actually just to `user_attributes: access_token.user_attributes`